src/Diagnostics.DataProviders/DataProviders/MdmDataProvider.cs

```diff
@@ -206,5 +210,53 @@ public async Task<IEnumerable<DataTable>> GetMultipleTimeSeriesAsync(DateTime st
 
             return result;
         }
+
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Reliability", "CA2007:Consider calling ConfigureAwait on the awaited task", Justification = "<Pending>")]
+        public override async Task<HealthCheckResult> CheckHealthAsync(CancellationToken cancellationToken = default(CancellationToken))
+        {
+            HealthCheckResult result;
+            if (_configuration.HealthCheckInputs != null && _configuration.HealthCheckInputs.Any())
+            {
+                //namespace=Microsoft/Web/WebApps;sampling=average;metric=PrivateBytes
+                _configuration.HealthCheckInputs.TryGetValue("namespace", out string mdmNamespace);
```

Comment:
Is the `HealthCheckInputs` defined in config? I dont see it.

Reply:
Yes it is in the DataProviderConfigurationBase class. https://github.com/Azure/Azure-AppServices-Diagnostics/blob/master2/src/Diagnostics.DataProviders/DataProviderConfigurations/DataProviderConfigurationBase.cs#L23-L48

Reply:
My bad, I think your talking about the app settings right?

Reply:
Sounds good. thanks